src/Fantomas.Core/AstTransformer.fs

```diff
@@ -218,12 +218,11 @@ and visitSynExpr (synExpr: SynExpr) : TriviaNode list =
                        yield! Option.toList (mkNodeOption SynExpr_Paren_ClosingParenthesis rpr) |]
                 |> List.singleton
                 |> finalContinuation)
-        | SynExpr.Quote (operator, _, quotedSynExpr, _, range) ->
-            let continuations: ((TriviaNode list -> TriviaNode list) -> TriviaNode list) list =
-                [ visit operator; visit quotedSynExpr ]
-
-            processSequence finalContinuation continuations (fun nodes ->
-                mkSynExprNode SynExpr_Quote synExpr range (sortChildren [| yield! nodes |]))
+        | SynExpr.Quote (_, _, quotedSynExpr, _, range) ->
```

Comment:
When you only have one nested `SynExpr` it makes more sense to call `visit` directly.